setup.py

```diff
@@ -63,6 +63,7 @@ def run(self, *arg, **kw):
     'bcrypt',
     'boto',
     'CherryPy<8',  # see https://github.com/girder/girder/issues/1615
+    'jsonschema',
```

Comment:
Whoops, thought I had pushed my last commit onto this, turns out I did not actually commit the changes...